pandas/tests/groupby/test_groupby.py

```diff
@@ -840,7 +840,9 @@ def test_omit_nuisance(df):
 
     # won't work with axis = 1
     grouped = df.groupby({"A": 0, "C": 0, "D": 1, "E": 1}, axis=1)
-    msg = "reduction operation 'sum' not allowed for this dtype"
+    msg = "'DatetimeArray' does not implement reduction 'sum'"
```

Comment:
I suppose we should add/fix that? Can you add a TODO comment?

Reply:
DatetimeArray isnt _supposed_ to implement sum

Reply:
still think this needs action/comment or can Resolve Conversation?

Reply:
Yes, this is fine, misread that it was only changing the already existing error message